### PR TITLE
 #18451349 user add products to their wishlist

### PR DESCRIPTION
**Pull request by Patienceineza**
#### What does this PR do?

This PR is adding more features on  user wishlist

#### Description of Task to be completed?

this task is associated with creating route for user to delete his/her wishlist, clear wishlist, getting all wishlists for all users

#### How should this be manually tested?

-clone the repo and run `npm run migarte:all`
-then login and put token in authorization 
-  use this endpoint `/wishlist/productId` to delete product from wishlist
- use this endpoint `/wishlist/all` to all products from wishlists for all users for this you have to be admin to access it
-  use this endpoint `/wishlist/clear` to clear your wishlist

#### Any background context you want to provide?

n/a

#### What are the relevant pivotal tracker/Trello stories?

https://www.pivotaltracker.com/story/show/184581349

#### Screenshots (if appropriate)

Screenshot 2023-03-23 at 21 53 37
Screenshot 2023-03-23 at 22 22 31

#### Questions:

n/a

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -452,7 +452,63 @@ you will get this response
 
 ```
 
-## adding product to cart
+
+## Remove a product from your wishlist 
+`post /wishlist/productID`
```
Comment (blackd44) [inline README.md]: i see all end path are mistaken to `POST` you can change them
all that are in this file

**Review by Martha-Iradukunda — APPROVED** [review]

**Review by blackd44 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Patienceineza) [reply README.md]: ok 

thank you

Comment (Patienceineza) [reply README.md]: I'm done  dalton

**Review by INValentin — APPROVED** [review]
LGTM

**Review by edwinnambaje — APPROVED** [review]
LGTM

**Review by blackd44 — APPROVED** [review]
LGTM

**Review by nshutijonathan — APPROVED** [review]
LGTM

**Pull request merged**